designs/virtualization/4.7/add-ip-addresses/add-ip-addresses.md

```diff
@@ -0,0 +1,34 @@
+---
+parent: Virtualization
+version: 4.7
+---
+# Add V4 and V6 IP Addresses
+
+## Adding a ‘Last restored’ column to the list view
+
+Currently we only show IPv4 IP addresses but with the addition of IPv6 we should consider how these strings should surface in the list/details view of the virtual machine.
+Currently we would not be able to show both types together though.
+
+## List view
+
+The column header will change to IP address(es) to match multiple addresses.
+
+![List view](img/VM-list.png)
+
+In cases where a Virtual Machine has more than one IP address, we will show the first IP address and the +X more action and add a popover that will include all the IP addresses for that VM.
+
+## V6 IP addresses are longer
+
+![V6 IP addresses](img/V6-names.png)
```

Comment:
for these longer IPv6 strings I wonder if we just did an ellipsis to avoid breaking the action to a second line. WDYT?